package.json

```
@@ -20,7 +20,7 @@
     "ocl": "./node_modules/.bin/cypress-tags run -eTAGS='@cypress and @ocl'",
     "microfrontend": "./node_modules/.bin/cypress-tags run -eTAGS='@cypress and @microfrontend'",
     "refapp3Login": "cypress run --spec src/test/resources/features/refapp-3.x/01-login/login.feature",
-    "refapp3Registration": "cypress run --spec src/test/resources/features/refapp-3.x/02-registration/registration.feature",
+    "refapp3SearchAndRegistration": "cypress run --spec src/test/resources/features/refapp-3.x/02-registration/search-and-registration.feature",
```

The issue is here. Please change this line to:

```suggestion
    "refapp3searchAndRegistration": "cypress run --spec src/test/resources/features/refapp-3.x/02-search-and-registration/search-and-registration.feature",
```